dace/frontend/fortran/intrinsics.py

```diff
@@ -714,20 +714,20 @@ def visit_Execution_Part_Node(self, node: ast_internal_classes.Execution_Part_No
                 initrange = i[0]
                 finalrange = i[1]
                 init = ast_internal_classes.BinOp_Node(
-                    lval=ast_internal_classes.Name_Node(name="tmp_parfor_" + str(self.count + range_index)),
+                    lval=ast_internal_classes.Name_Node(name="tmp_lbrt_" + str(self.count + range_index)),
```

Comment:
Same here - this is a base used by many intrinsics. Two different intrinsic replacements can emit the same `tmp_lbrt_0`, for example.

If this is a problem for name collision, then I will change the implementation to pass a custom prefix from each intrinsic.